### PR TITLE
Fix publish to NPM v2

### PR DESCRIPTION
**Pull request by mangs**
**Pull Request Checklist**

- [x] I have read the [CONTRIBUTING](/CONTRIBUTING.md) document
- [ ] Readme and changelog updates were made reflecting this PR's changes
- [ ] Increase the project version number in [`package.json`](/package.json) and [`package-lock.json`](/package-lock.json) following [Semantic Versioning](http://semver.org/)

**Changes Included**

- Fix publish to NPM v2

### CODE REVIEW

File: .github/workflows/publishWorkflow.yml [hunk 1]
```diff
@@ -11,12 +11,11 @@ jobs:
       - uses: actions/setup-node@v4
         with:
           node-version: "20.10.0"
+          registry-url: https://registry.npmjs.org
       - id: super-cache
         uses: mangs/super-cache-action@v3
       - if: steps.super-cache.outputs.cache-hit != 'true'
         run: npm ci
-        env:
-          NODE_AUTH_TOKEN: ${{ secrets.PACKAGES_TOKEN }}
```
Comment (mangs) [inline .github/workflows/publishWorkflow.yml:19]: We're not installing any private packages

File: .github/workflows/publishWorkflow.yml [hunk 2]
```diff
@@ -11,12 +11,11 @@ jobs:
       - uses: actions/setup-node@v4
         with:
           node-version: "20.10.0"
+          registry-url: https://registry.npmjs.org
```
Comment (mangs) [inline .github/workflows/publishWorkflow.yml:14]: This is probably why publishing wasnt working

**Review by mangs — COMMENTED** [review]
Some notes for context

File: .github/workflows/publishWorkflow.yml (hunk 2 above)
Comment (jduthon) [reply .github/workflows/publishWorkflow.yml:14]: Ohhh interesting!
I'd have imagined this to be the default!

**Review by jduthon — APPROVED** [review]
🤞

File: .github/workflows/publishWorkflow.yml (hunk 2 above)
Comment (mangs) [reply .github/workflows/publishWorkflow.yml:14]: That's what I assumed but I was wrong

**Pull request merged**